app/views/preferences/index.html.erb

```diff
@@ -1,27 +1,38 @@
-<div id="display_box">
+<div class="bootstrap">
+  <div class="container-fluid">
```

Comment:
Maybe indent this block at the same level as `.bootstrap`? I know the indentation will be "wrong" but it will allow us to remove the `div.bootstrap` later without extraneous whitespace changes.